src/referencePackages/src/microsoft.extensions.dependencymodel/2.1.0/Microsoft.Extensions.DependencyModel.2.1.0.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net451;netstandard1.3;netstandard1.6</TargetFrameworks>
```

Comment:
Can you remove this net451 so this PR does not cause conflicts with https://github.com/dotnet/source-build-reference-packages/pull/596